sherpa/astro/flux.py

```diff
@@ -97,8 +98,8 @@ def simulated_pars_within_ranges(mysamples, mysoftmins, mysoftmaxs):
 
     def print_sample_result(title, arg):
 
-        print '%s = %g, + %g, - %g' % (title, arg[0], arg[1] - arg[0],
-                                       arg[0] - arg[2])
+        print('%s = %g, + %g, - %g' % (title, arg[0], arg[1] - arg[0],
+                                       arg[0] - arg[2]))
```

Comment:
Is it worth changing

    fmt % vals

to

    fmt.format(vals)

[with appropriate changes like %<xx> -> {<yy>} in `fmt`]? I had thought (a long while ago) that the % syntax was deprecated but I'm a lot-less sure now.